tests-e2e/cypress/integration/playbooks/edit_spec.js

```diff
@@ -1195,5 +1195,240 @@ describe('playbooks > edit', () => {
                 });
             });
         });
+
+        describe('when a new member joins the channel', () => {
+            describe('add the channel to sidebary category', () => {
+                it('is disabled in a new playbook', () => {
+                    // # Visit the selected playbook
+                    cy.visit(`/playbooks/playbooks/${testPlaybook.id}/edit`);
+
+                    // # Switch to Actions tab
+                    cy.get('#root').findByText('Actions').click();
```

Comment:
```suggestion
                    cy.findByText('Actions').click();
```
I don't think the `get('#root')` is necessary on any of these. This `visit -> switch to 'Actions'` sequence also happens at the start of all the `it` blocks, so it could probably go in a `beforeEach` for the parent `describe` block.